### PR TITLE
Update Studio MD files

### PR DESCRIPTION
**Pull request by junmoy**
Update Studio MD files

### CODE REVIEW

File: docs/tizen-studio/native-tools/ui-builder-n.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
...
 
-The native UI Builder is not only a UI layout code generating tool. It also supports an easy-to-use programming model, allows you to [support multiple screens](multiple-screen-n.md) within a single application, and makes it possible to [manage various data sources through databinding](ui-builder-databind-n.md). If you have an existing EFL UI project, you can [convert it to a UI Builder project](ui-builder-conversion-n.md) to ease the task of expanding the application and optimizing it to new devices.
+The native UI Builder is not only a UI layout code generating tool. It also supports an easy-to-use programming model, allows you to [support multiple screens](multiple-screen_n.md) within a single application, and makes it possible to [manage various data sources through databinding](ui-builder-databind-n.md). If you have an existing EFL UI project, you can [convert it to a UI Builder project](ui-builder-conversion-n.md) to ease the task of expanding the application and optimizing it to new devices.
```
Comment (journey2w) [inline docs/tizen-studio/native-tools/ui-builder-n.md]: Why did you change multiple-screen-n.md?

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md [hunk 2]
```diff
@@ -13,4 +13,5 @@ To develop IoT applications with SmartThings&trade;:
 
 - [Create a SmartThings&trade; project](rt-create-smartthings.md).
 - [Build the SmartThings&trade; project](rt-build-smartthings.md).
+- [Manage SmartThings™ certificates](rt-certificate-smartthings.md).
```
Comment (journey2w) [inline docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: Please change to SmartThings&trade;.

File: docs/tizen-studio/native-tools/ui-builder-n.md (hunk 1 above)
Comment (junmoy) [reply docs/tizen-studio/native-tools/ui-builder-n.md]: It should be my mistake. I'll fix it.

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md (hunk 2 above)
Comment (junmoy) [reply docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: Sure!

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
-# SmartThings&trade; Extension
+# SmartThings™ Extension
 
 ## Dependencies
 - Ubuntu Only
 
-The Tizen Studio for RT provides many extended features for SmartThings&trade;-connected IoT applications.
+The Tizen Studio for RT provides many extended features for SmartThings™-connected IoT applications.
```
Comment (journey2w) [inline docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: please change from SmartThings™ to SmartThings\&trade;.

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
-# SmartThings&trade; Extension
+# SmartThings™ Extension
```
Comment (journey2w) [inline docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: please change from SmartThings™ to SmartThings\&trade;.

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md [hunk 5]
```diff
@@ -1,17 +1,19 @@
...
 
 You can select a device model template from the Project Wizard and create a project that, by default , contains a lot of stub code for easy development. You only need to add user-defined logic or hardware-specific I/O control.
 
 The Tizen Studio for RT supports the [OCF](https://openconnectivity.org) (Open Connected Foundation) resource model. You can view and modify the resource model in the Model Manager.
 
 To develop IoT applications with SmartThings&trade;:
 
-- [Create a SmartThings&trade; project](rt-create-smartthings.md).
-- [Build the SmartThings&trade; project](rt-build-smartthings.md).
+- [Create a SmartThings™ project](rt-create-smartthings.md).
```
Comment (journey2w) [inline docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: Please change from SmartThings™ to SmartThings\&trade;.

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md [hunk 6]
```diff
@@ -1,17 +1,19 @@
...
+- [Build the SmartThings™ project](rt-build-smartthings.md).
```
Comment (journey2w) [inline docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: please change from SmartThings™ to SmartThings\&trade;.

File: docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md (hunk 2 above)
Comment (journey2w) [reply docs/tizen-studio/rt-sdk/rt-stthings-extensions-cover.md]: I'm sorry for letting you be confused. GitHub displays \&trade; to &trade;.
I mean you change from SmartThings™ to SmartThings\&trade;, so I left a comment only here.
